pwiz_tools/Skyline/FileUI/PeptideSearch/TransitionSettingsControl.cs

```diff
@@ -217,6 +222,10 @@ public int MaxIonMz
 
         public void Initialize(ImportPeptideSearchDlg.Workflow workflow)
         {
+            // Reset control locations, in case this isn't the first call to Initialize.
+            foreach (Control control in Controls)

```

Comment:
I would have written this as:
foreach (var keyValuePair in _originalLocations) { keyValuePair.Key.Location = keyValuePair.Value }
That way you don't have to worry about what if the control is missing from the dictionary.